[ERROR] server - Error: connect ETIMEDOUT 5.75.249.160:443

<!-- IMPORTANT: Please disable plugins prior to posting a bug report.  If you have a problem with a plugin please post on the plugin repository.  Thanks! -->

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Server (please complete the following information):**
 - Etherpad version:
 - OS: [e.g., Ubuntu 20.04]
 - Node.js version (`node --version`):
 - npm version (`npm --version`):
 - Is the server free of plugins: 

**Desktop (please complete the following information):**
 - OS: [e.g. iOS]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - Browser [e.g. stock browser, safari]
 - Version [e.g. 22]

**Additional context**
Add any other context about the problem here.

> Hi @fuwens . Could you please give a more detailed explanation about the error? I have no idea what this ip address is

After I started the project, I opened the plug-in page, and the service hung up a moment later，Console error：
`[2023-11-29T16:28:26.562] [ERROR] server - Metrics at time of fatal error:
{
  "httpStartTime": 1701244518792,
  "memoryUsage": 281792512,
  "memoryUsageHeap": 78908416,
  "ueberdb_lockAwaits": 0,
  "ueberdb_lockAcquires": 36,
  "ueberdb_lockReleases": 36,
  "ueberdb_reads": 29,
  "ueberdb_readsFailed": 0,
  "ueberdb_readsFinished": 29,
  "ueberdb_readsFromCache": 23,
  "ueberdb_readsFromDb": 6,
  "ueberdb_readsFromDbFailed": 0,
  "ueberdb_readsFromDbFinished": 6,
  "ueberdb_writes": 9,
  "ueberdb_writesFailed": 0,
  "ueberdb_writesFinished": 8,
  "ueberdb_writesObsoleted": 0,
  "ueberdb_writesToDb": 8,
  "ueberdb_writesToDbFailed": 0,
  "ueberdb_writesToDbFinished": 8,
  "ueberdb_writesToDbRetried": 0,
  "totalUsers": 1,
  "activePads": 1,
  "httpRequests": {
    "meter": {
      "mean": 0.20063278507249493,
      "count": 18,
      "currentRate": 0.20063260397879576,
      "1MinuteRate": 0.15776642210720937,
      "5MinuteRate": 0.05213757442288715,
      "15MinuteRate": 0.019072227426553925
    },
    "histogram": {
      "min": 0.29988300800323486,
      "max": 249.43932600319386,
      "sum": 289.50200203061104,
      "variance": 3412.1039842923833,
      "mean": 16.08344455725617,
      "stddev": 58.41321754784942,
      "count": 18,
      "median": 0.5741105005145073,
      "p75": 2.5432819984853268,
      "p95": 249.43932600319386,
      "p99": 249.43932600319386,
      "p999": 249.43932600319386
    }
  },
  "connects": {
    "mean": 0.0642768483954513,
    "count": 5,
    "currentRate": 0.06427684996211616,
    "1MinuteRate": 0.04049182241086165,
    "5MinuteRate": 0.014264678977627315,
    "15MinuteRate": 0.005271523522343614
  },
  "lastDisconnect": 1701246506562,
  "disconnects": {
    "mean": 0.05528771880527406,
    "count": 4,
    "currentRate": 0.05528771962295572,
    "1MinuteRate": 0.030760842896421996,
    "5MinuteRate": 0.011351327055928043,
    "15MinuteRate": 0.004210840360390726
  }
}
[2023-11-29T16:28:26.562] [ERROR] server - Error: connect ETIMEDOUT 5.75.249.160:443
    at AxiosError.from (/xxxxx/src/node_modules/axios/dist/node/axios.cjs:837:14)
    at RedirectableRequest.handleRequestError (/xxxxx/src/node_modules/axios/dist/node/axios.cjs:3029:25)
    at RedirectableRequest.emit (node:events:513:28)
    at eventHandlers.<computed> (/xxxxx/src/node_modules/follow-redirects/index.js:14:24)
    at ClientRequest.emit (node:events:513:28)
    at TLSSocket.socketErrorListener (node:_http_client:502:9)
    at TLSSocket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
[2023-11-29T16:28:26.562] [INFO] server - Exiting...
[2023-11-29T16:28:26.563] [INFO] server - Stopping Etherpad...
[2023-11-29T16:28:26.563] [INFO] http - Closing HTTP server...
[2023-11-29T16:28:26.564] [INFO] socket.io - Closing socket.io engine...
[2023-11-29T16:28:26.564] [INFO] access - [LEAVE] pad:1 socket:voRttdoDRcgoYCxTAAAE IP:127.0.0.1 authorID:a.N0WXTNyii9Zu5Z5I username:admin
[2023-11-29T16:28:26.565] [INFO] socket.io - All socket.io clients have disconnected
[2023-11-29T16:28:26.569] [INFO] http - Waiting for 3 HTTP clients to disconnect...
[2023-11-29T16:28:26.670] [INFO] ueberDB - Database closed
[2023-11-29T16:28:26.671] [ERROR] socket.io - Error handling pad message from voRttdoDRcgoYCxTAAAE: Error: client disconnected
    at exports.handleMessage (/xxxxx/src/node/handler/PadMessageHandler.js:327:54)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /xxxxx/src/node/handler/SocketIORouter.js:82:14
[2023-11-29T16:28:27.583] [INFO] http - HTTP server closed
[2023-11-29T16:28:27.584] [INFO] server - Etherpad stopped
[2023-11-29T16:28:27.584] [INFO] server - Waiting for Node.js to exit...
[2023-11-29T16:28:32.585] [ERROR] server - Something that should have been cleaned up during the shutdown hook (such as a timer, worker thread, or open connection) is preventing Node.js from exiting
[2023-11-29T16:28:32.585] [ERROR] server - Enable `dumpOnUncleanExit` setting to get a dump of objects preventing a clean exit
[2023-11-29T16:28:32.585] [ERROR] server - Forcing an unclean exit...
 `
Plugin cannot be installed！

--- a/src/static/js/pluginfw/installer.js
+++ b/src/static/js/pluginfw/installer.js
@@ -15,6 +15,10 @@ const onAllTasksFinished = async () => {
   await hooks.aCallAll('restartServer');
 };
 
+const headers = {
+  'User-Agent': 'Etherpad/' + settings.getEpVersion(),
+}
+
 let tasks = 0;
 
 const wrapTaskCb = (cb) => {
@@ -77,12 +81,15 @@ exports.getAvailablePlugins = (maxCacheAge) => {
           return resolve(exports.availablePlugins);
       }
 
-      await axios.get('https://static.etherpad.org/plugins.json')
-          .then(pluginsLoaded => {
-              exports.availablePlugins = pluginsLoaded.data;
-              cacheTimestamp = nowTimestamp;
-              resolve(exports.availablePlugins);
-          })
+      await axios.get('https://static.etherpad.org/plugins.json', {headers: headers})
+        .then(pluginsLoaded => {
+            exports.availablePlugins = pluginsLoaded.data;
+            cacheTimestamp = nowTimestamp;
+            resolve(exports.availablePlugins);
+        })
+        .catch(async err => {
+          return reject(err);
+        });
   })
 }
 

